### PR TITLE
add bluez to dockerfile for use in bluetooth tests

### PR DESCRIPTION
**Pull request by rcooke-warwick**

### CODE REVIEW

File: core/Dockerfile.template [hunk 1]
```diff
@@ -10,10 +10,11 @@ RUN npm ci
 FROM balenalib/%%BALENA_MACHINE_NAME%%-node:10-buster-build
 
 ENV UDEV=1
+ENV DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
```
Comment (vipulgupta2048) [inline core/Dockerfile.template:13]: hey @rcooke-warwick I recently read this https://github.com/balena-io/docs/issues/591, any side-effects you know that can or did happen with having the DBUS path as env var

**Review by vipulgupta2048 — COMMENTED** [review]
LGTM

File: core/Dockerfile.template (hunk 1 above)
Comment (rcooke-warwick) [reply core/Dockerfile.template:13]: @vipulgupta2048 I haven't seen any side effects yet! and have been running commands from the container

**Review by vipulgupta2048 — APPROVED** [review]
Merging @rcooke-warwick

**Pull request merged**